front/pages/w/[wId]/assistant/labs/transcripts/index.tsx

```diff
@@ -359,6 +381,42 @@ export default function LabsTranscriptsIndex({
     return updateIsActive(transcriptConfigurationId, isActive);
   };
 
+  const getDefaultConfiguration = async (provider: string) => {
+    const response = await fetch(
```

Comment:
Let's follow the recent pattern, fetch should live alongside swr hooks in `/swr`. Also why not using a hook here?